openupgrade_scripts/scripts/base/14.0.1.3/post-migrate.py

```diff
@@ -71,9 +73,27 @@ def fix_module_category_parent_id(env):
     )
 
 
+def delete_module_not_exist(env):
+    env.cr.execute("SELECT id, name from ir_module_module")
+    module_list = env.cr.dictfetchall()
+    module_delete = []
+    for module in module_list:
```

Comment:
@hoangtiendung070797 đặt tên biến cho chuyên nghiệp tý nhé
```suggestion
    module_list = env.cr.dictfetchall()
    module_delete = []
    for module in module_list:
```
```suggestion
    modules_to_delete = []
    for module in env.cr.dictfetchall():
```

Reply:
@royleviindoo dạ vâng anh